sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py

```diff
@@ -277,6 +277,9 @@ def begin_recognize_custom_forms(self, model_id, form, **kwargs):
                 :caption: Recognize fields and values from a custom form.
         """
 
+        if not model_id:
```

Comment:
Curious - don't we have match conditions for delete operations in form?

Reply:
Can you show me where you see that?